lib/quadquizaminos_web/live/contests/components/contest_component.ex

```diff
@@ -13,7 +13,7 @@ defmodule QuadquizaminosWeb.ContestsLive.ContestComponent do
 
   @impl true
   def mount(socket) do
-    {:ok, socket |> assign(editing_date?: false, rsvped?: false)}
+    {:ok, socket |> assign(editing_date?: false, rsvped?: false, modal: false)}
```

Comment:
modal appears  on the parent LiveView(contest_live), I don't think its necessary to add `modal: false` at this point